karavan-app/src/main/webui/src/projects/ProjectsPage.tsx

```diff
@@ -80,7 +83,23 @@ export class ProjectsPage extends React.Component<Props, State> {
     }
 
     onProjectDelete = (project: Project) => {
-        this.setState({isDeleteModalOpen: true, projectToDelete: project})
+        KaravanApi.getProjectPipelineStatus(project.projectId, this.props.config.environment, (status?: PipelineStatus) => {
+            if(status === undefined){
+                this.setState({isDeleteModalOpen: true, projectToDelete: project})
+            }else if(status?.result ==="Running"){
+                this.setState({isPodRunningModalOpen: true, projectToDelete: project})
+            }else if(status?.result ==="Succeeded"){
```

Comment:
What if status.result is not "Succeeded"? Ex. latest pipeline run crashed, but previous was succeeded and Deployment/Pod is Up and Running

Reply:
Should the new pipeline run not delete the old Deployment first?

Reply:
Pipeline replace existing deployment if this pipeline successfully finished

Reply:
Implemented the following checks:           
  * If the pipeline is Running or Started, ask the user to delete the Deployment first.
  * If any other status of the Pipeline or undefined, Check the deployment status.
  * If deployment is undefined, Show the user, the delete Modal dialog,
  * If deployment is there, the ask user to delete the deployment first.